pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -177,11 +177,14 @@ stdenv.mkDerivation rec {
     patchShebangs .
   '';
 
+  requiredSystemFeatures = [ "big-parallel" ];
+
   meta = {
     description = "Web content rendering engine, GTK port";
     homepage = "https://webkitgtk.org/";
     license = licenses.bsd2;
     platforms = platforms.linux ++ platforms.darwin;
     maintainers = teams.gnome.members;
+    timeout = 172800; # 48 hours (increased from the Hydra default of 10h)
```

Comment:
Nit: also 48 * 3600? To keep the code expressive instead of relying on correct comments.

Reply:
Could be.  I just copied the whole line from chromium.

Reply:
Should we even let it build so long? Even on my laptop it does not take more than a few hours so maybe if a builder takes so long, we should just put it out of its misery.

Reply:
I don't like the (repeated) failures for two reasons:
- no nice way of restarting all the corresponding propagated failures (without restarting others), and
- the interruptions seem to waste resources (assuming the build isn't really stuck but there are some other limits meant for that).

Still, I expect that the `big-parallel` tag would suffice on its own.

Reply:
I dropped the `meta.timeout` changes for now, as they seemed far less important than the other part anyway.